packages/backend/src/modules/user/user.mapper.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-underscore-dangle, class-methods-use-this */
+import { UserDto } from 'shared'
+
+import { User } from './user'
+
+export class UserMapper {
```

Comment:
use interface implements Mapper<User, UserDTO>

Reply:
/home/michal/dev/priv/camp-trello-clone/packages/backend/src/application/Mapper.ts

Reply:
/home/michal/dev/priv/camp-trello-clone/packages/backend/src/application/Mapper.ts

Reply:
@mcwiekala rather should be only ```/camp-trello-clone/packages/backend/src/application/Mapper.ts``` because ```/home/michal/dev/priv/``` - it is a part of path in your OS 😄